.circleci/config.yml

```diff
@@ -175,6 +175,37 @@ jobs:
           json: build/<<parameters.slug>>.$BUILD_VERSION.json
           version: $BUILD_VERSION
 
+  package-build:
+    docker:
+      - image: cimg/node:lts
+    working_directory: ~/project/packages/headless
+    steps:
+      - attach_workspace:
+          at: ~/project
+      - run:
+          name: NPM package install and build
+          command: |
+            npm install
+            npm run build
+      - persist_to_workspace:
+          root: ~/project
+          paths:
+            - packages/headless
+
+  package-test:
+    docker:
+      - image: cimg/node:lts
+    working_directory: ~/project/packages/headless
+    steps:
+      - attach_workspace:
+          at: ~/project
+      - run:
+          name: NPM package tests
+          command: |
+            npm install
```

Comment:
Do you have to run `npm install` again? Looks like it runs in `package-build` and is persisted to the workspace, and then `package-test` attaches to the same workspace.

Reply:
I did. The test failed even when the workspace was persisted.  I think I'm going to try and find a node/npm command, if there is one, that does the package.json caching, for the performance boost.

Reply:
Maybe there's something about the npm install that sets something up in the environment. #still-learning 
There is https://circleci.com/developer/orbs/orb/circleci/node.  circleci/node and takes care of caching https://circleci.com/developer/orbs/orb/circleci/node#commands-install-packages.  I'm gonna try that.